Formula/mitre-saf.rb

```diff
@@ -0,0 +1,25 @@
+require "language/node"
+
+class MitreSaf < Formula
+  desc "CLI for streamlining security automation of systems and DevOps pipelines"
+  homepage "https://github.com/mitre/saf"
+  url "https://registry.npmjs.org/@mitre/saf/-/saf-1.1.11.tgz"
+  sha256 "d93b47b083835db60c3c2473aa50e6a71322efb45a76083c2f0a3e5452d0287e"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+  end
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"saf", "--help"
+    assert_match "@mitre/saf/#{version}", shell_output("#{bin}/saf --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.